src/opcodes/tests/test_constants.cairo

```diff
@@ -81,6 +81,75 @@ fn test_op_data_all() {
     test_op_data(4);
     test_op_data(5);
     test_op_data(6);
+    test_op_data(7);
```

Comment:
Could you change this `test_op_data_all` function to use a loop instead of repeating this for each number?